src/components/project/moreProjects.tsx

```diff
@@ -1 +1,196 @@
-export {};
+import { Projects } from "../../interfaces/project";
+import { IconButton } from "../global/iconButton";
+import styled from "@emotion/styled";
+import { breakpoints, dimensions } from "../../styles/variables";
+import { getRemSize } from "../../styles/globalCss";
+import useIsDesktop from "../../hooks/useIsDesktop";
+import { Fragment } from "react";
+import Image from "next/image";
+
+interface IProjectsData {
+  projects: Projects;
+}
+
+const StyledHeading = styled.h2`
+  font-weight: 500;
+  text-align: center;
+  width: 100%;
+  margin-bottom: 60px;
+  margin-left: auto;
+  margin-right: auto;
+  font-size: ${getRemSize(dimensions.headingSizes.medium.desktop)};
+
+  @media (max-width: ${breakpoints.md}px) {
+    font-size: ${getRemSize(dimensions.headingSizes.medium.desktop)};
+    margin-bottom: 45px;
+  }
+
+  @media (max-width: ${breakpoints.sm}px) {
+    font-size: ${getRemSize(dimensions.headingSizes.large.mobile)};
+    margin-bottom: 32px;
+  }
+`;
+
+const StyledProjectCardsWrapper = styled.div`
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+  width: 100%;
+
+  @media (max-width: ${breakpoints.md}px) {
+    flex-direction: column;
+  }
+`;
+const StyledProjectCard = styled.div`
+  max-width: 680px;
+  min-width: 200px;
+  width: 100%;
+  height: 264px;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  position: relative;
+  & img {
```

Comment:
There's actually a little bit of filter on the image on figma making it slightly darker (black 30%) - you can apply filters to images in css, but if you have a question let me know